site/book/06-deploying-packages/00.md

```diff
@@ -0,0 +1,68 @@
+In the last chapter of this book, we are going to cover how you deploy a kpt
+package to a Kubernetes cluster and how the cluster state is managed as the
+package evolves over time.
+
+We use `kpt live apply` instead `kubectl apply` since it provides some critical
+functionality not provided by the latter: pruning and reconciled status. To
+enable this functionality, we need a cluster-side mechanism for grouping and
+tracking resources belonging to a package. This cluster-side grouping is
+implemented using a custom resource of kind `ResourceGroup`. Otherwise,
+`kpt live` and `kubectl` are complementary. For example, you can use
+`kubectl get` as normal would.
+
+## Pruning
+
+`live apply` will automatically delete cluster resources that are no longer
+present in the local package. This clean-up functionality is called pruning.
+
+For example, consider a package which has been applied with the following three
+resources:
+
+```
+service-1 (Service)
+deployment-1 (Deployment)
+config-map-1 (ConfigMap)
+```
+
+Then imagine the package is updated to contain the following resources:
+
+```
+service-1 (Service)
+deployment-1 (Deployment)
+config-map-2 (ConfigMap)
+```
+
+When the updated package is applied, `config-map-1` is automatically pruned from
```

Comment:
Maybe use removed/deleted instead of pruned here just to avoid explaining pruning by saying resources are pruned.